iosHyperskillApp/iosHyperskillApp/Sources/Modules/StepQuiz/Views/StepQuizView.swift

```diff
@@ -68,19 +65,11 @@ struct StepQuizView: View {
 
             ScrollView {
                 VStack(alignment: .leading, spacing: appearance.interItemSpacing) {
-                    StepQuizStatsView(text: viewData.formattedStats)
-
                     if case .unsupported = viewData.quizType {
                         StepQuizStatusView(state: .unsupportedQuiz)
                     }
 
-                    StepTextView(
-                        text: viewData.stepText,
-                        appearance: .init(
-                            textFont: appearance.stepTextFont,
-                            textColor: appearance.stepTextColor
-                        )
-                    )
+                    StepTextView(text: viewData.stepText)
```

Comment:
Search for usages of the `StepTextView` - we also need to update rendering at the `StepQuizCodeFullScreenDetailsView`